girder_annotation/girder_large_image_annotation/models/annotationelement.py

```diff
@@ -374,6 +415,23 @@ def _boundingBox(self, element):
             bbox['highx'] = max(x0, x1)
             bbox['highy'] = max(y0, y1)
             bbox['details'] = len(element['values'])
+        elif element.get('type') == 'imageoverlay':
+            lowx = highx = lowy = highy = highz = 0
+            try:
+                lowx, highx, lowy, highy = Annotationelement()._overlayBounds(element)
+                baseImageItem = ImageItem().load(annotation['itemId'], force=True)
+                baseImageMetadata = ImageItem().getMetadata(baseImageItem)
+                highz = baseImageMetadata.get('levels', 0)
```

Comment:
In thinking about this, I don't think we want to store levels in the z bounding box -- the levels aren't conceptually heights (whereas there ARE multi-frame images that have a z-stack of frames).  I think for now, the z will always just be zero.  If we do that, then we don't have to pass the parent annotation into the `_boundingBox` method.

Reply:
See 95cf345